src/transport/PairingSession.h

```diff
@@ -36,6 +36,28 @@ class DLL_EXPORT PairingSession
     PairingSession() {}
     virtual ~PairingSession() {}
 
+    NodeId GetPeerNodeId() const { return mPeerNodeId; }
```

Comment:
Please document the semantics.  When can this return `kUndefinedNodeId`?  What does that return value mean?

Reply:
Replaced with optional